package.json

```diff
@@ -9,17 +9,19 @@
   "engines": {
     "node": "18.x"
   },
+  "type": "module",
   "scripts": {
     "develop": "gatsby develop",
-    "start": "gatsby develop",
+    "start": "ts-node --experimentalSpecifierResolution node download-data.ts & gatsby develop",
```

Comment:
Why are you not calling `download-data` here?

Reply:
done